src/Backend/Modules/MediaLibrary/Installer/Data/locale.xml

```diff
@@ -514,6 +514,10 @@
         <translation language="en"><![CDATA[You are required to connect media.]]></translation>
         <translation language="nl"><![CDATA[Je bent verplicht om media te koppelen.]]></translation>
       </item>
+      <item type="label" name="Select">
```

Comment:
select what?

Reply:
I think this can be a generic label. But I can always change this to `SelectItem`.